### ISSUE TITLE
Header

### ISSUE THREAD

**Issue by KrooshalUX** [opensearch-project/oui#110]

**Comment by KrooshalUX** [comment]
I have an open PR for this but not sure how to move this forward as I am blocked on making further commits @BSFishy

**Comment by KrooshalUX** [comment]
Blocked pending help with stalled PR

**Comment by anirudha** [comment]
Closing — this documentation issue has been open since 2022/2023 with no progress. The doc site has been significantly updated since then. If specific documentation gaps remain, please open a new issue with current context.
